man/nvim.1

```diff
@@ -345,6 +351,11 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev VIM
+Used to locate various user files, such as the user-local nvimrc.
```

Comment:
I think just "user vimrc" is a bit better (didn't find the expression "user-local vimrc" anywhere else in the repo, and it feels weird to me).